src/pages/networks/NetworkDetailOverview.tsx

```diff
@@ -96,12 +98,17 @@ const NetworkDetailOverview: FC<Props> = ({ network }) => {
           </table>
         </Col>
       </Row>
-      {!isPhysicalManagedNetwork && (
-        <Row className="section">
-          <Col size={3}>
-            <h2 className="p-heading--5">Status</h2>
-          </Col>
-          <Col size={7}>
+      <Row className="section">
+        <Col size={3}>
+          <h2 className="p-heading--5">Status</h2>
+        </Col>
+        <Col size={7}>
+          {isNetworkStateError && (
```

Comment:
small nit: why not use a ternary here with `isNetworkStateError`?

Reply:
I prefer the more verbose && and repeating the condition style here. Because the first conditional block has more than one or two lines. It makes the second conditional block easier to read and to understand. The condition of the ternary is so many lines up, and you need to keep it in mind to understand the condition for the else block. Repeating the condition tries to solve this.

Reply:
sounds good :+1: